### PR TITLE
Refactored to no longer need treasury

### PR DESCRIPTION
**Pull request by mostarz93**
There are 2 tests that are acting strange:

In Spigot.t.sol:
 - test_claimOwnerTokens_AsOwner

In SpigotedLine.t.sol
 - test_sweep_to_borrower_when_repaid -> has an assert that im not sure what the point is.

### CODE REVIEW

**Review by kibagateaux — COMMENTED** [review]
Pretty sure there should still be a `+ overflow` at the end here

File: contracts/utils/SpigotLib.sol [hunk 1]
```diff
@@ -213,8 +222,8 @@ library SpigotLib {
     }
 
     /** see Spigot.getEscrowed*/
-    function getEscrowed(SpigotState storage self, address token) external view returns (uint256) {
```
Comment (kibagateaux) [inline contracts/utils/SpigotLib.sol:216]: i dont see `getOperatorTokens` here

Comment (kibagateaux) [reply contracts/utils/SpigotLib.sol:216]: actually we dont even use this function in Spigot.sol. you can delete it

Comment (mostarz93) [reply contracts/utils/SpigotLib.sol:216]: this must be stale bc i dont have this func on my local branch

**Pull request merged**